.travis.yml

```diff
@@ -1,3 +1,13 @@
-install: gem install kramdown
+language: python
+python:
+  - "2.7"
+sudo: false
+install:
+  - gem install kramdown
 
-script: make
+  - git clone --depth=1 --branch=master https://github.com/tabatkins/bikeshed.git ./bikeshed
+  - pip install pygments
+  - pip install --editable ./bikeshed
+  - bikeshed update
+script:
+  make
```

Comment:
I think you're missing a `bikeshed ...` line of some sort here because the bikeshed version of the spec isn't being built: https://travis-ci.org/w3c/webappsec-subresource-integrity/builds/129305471